### ISSUE TITLE
pyodide_build buildpkg does not produce a .js file and does not report an error

### ISSUE THREAD

**Issue by paulftw** [pyodide/pyodide#2147]
## 🐛 Bug

I'm following official instructions on [creating a package](https://pyodide.org/en/stable/development/new-packages.html#building-the-package-and-investigating-issues) to build cadquery from github repo.

### To Reproduce

1. Build pyodide using official docker, install pyodide_build
2. Create `packages/cadquery/meta.yaml`:
```
package:
  name: cadquery
  version: "2.1"
source:
  url: https://github.com/CadQuery/cadquery/archive/refs/heads/master.zip

requirements:
  run:
    - ocp 7.5.3
    - hdf5 1.10.6 *1114
    - pyparsing >=2.1.9
    - ezdxf
    - ipython
    - typing_extensions
    - nptyping
    - nlopt
    - multimethod 1.6

test:
  imports:
    - cadquery

```

3. Run commands from documentation: 
```
export PYTHONPATH="$PYTHONPATH:/Users/myname/my-pyodide/pyodide-build/"
python -m pyodide_build buildpkg meta.yaml
cp build/*.data build/*.js ../../build/
```

Output of `cp`:
```
$ cp build/*.data build/*.js ../../build/
cp: build/*.data: No such file or directory
cp: build/*.js: No such file or directory
```

### Expected behavior

I expected to either get an error printed or a js file generated (or a mention in the documentation on why js isn't produced sometimes.)

Other details:

Output of `pyodide_build`:
```
$ python -m pyodide_build buildpkg meta.yaml
[2022-01-30 13:11:28] Building package cadquery...
running build
running build_py
creating build
creating build/lib
creating build/lib/cadquery
copying cadquery/cq_directive.py -> build/lib/cadquery
copying cadquery/units.py -> build/lib/cadquery
copying cadquery/cq.py -> build/lib/cadquery
copying cadquery/__init__.py -> build/lib/cadquery
copying cadquery/hull.py -> build/lib/cadquery
copying cadquery/selectors.py -> build/lib/cadquery
copying cadquery/types.py -> build/lib/cadquery
copying cadquery/cqgi.py -> build/lib/cadquery
copying cadquery/utils.py -> build/lib/cadquery
copying cadquery/assembly.py -> build/lib/cadquery
copying cadquery/sketch.py -> build/lib/cadquery
creating build/lib/cadquery/plugins
copying cadquery/plugins/__init__.py -> build/lib/cadquery/plugins
creating build/lib/cadquery/contrib
copying cadquery/contrib/__init__.py -> build/lib/cadquery/contrib
creating build/lib/cadquery/occ_impl
copying cadquery/occ_impl/sketch_solver.py -> build/lib/cadquery/occ_impl
copying cadquery/occ_impl/shapes.py -> build/lib/cadquery/occ_impl
copying cadquery/occ_impl/solver.py -> build/lib/cadquery/occ_impl
copying cadquery/occ_impl/geom.py -> build/lib/cadquery/occ_impl
copying cadquery/occ_impl/jupyter_tools.py -> build/lib/cadquery/occ_impl
copying cadquery/occ_impl/__init__.py -> build/lib/cadquery/occ_impl
copying cadquery/occ_impl/assembly.py -> build/lib/cadquery/occ_impl
creating build/lib/cadquery/occ_impl/importers
copying cadquery/occ_impl/importers/__init__.py -> build/lib/cadquery/occ_impl/importers
copying cadquery/occ_impl/importers/dxf.py -> build/lib/cadquery/occ_impl/importers
creating build/lib/cadquery/occ_impl/exporters
copying cadquery/occ_impl/exporters/amf.py -> build/lib/cadquery/occ_impl/exporters
copying cadquery/occ_impl/exporters/__init__.py -> build/lib/cadquery/occ_impl/exporters
copying cadquery/occ_impl/exporters/dxf.py -> build/lib/cadquery/occ_impl/exporters
copying cadquery/occ_impl/exporters/utils.py -> build/lib/cadquery/occ_impl/exporters
copying cadquery/occ_impl/exporters/svg.py -> build/lib/cadquery/occ_impl/exporters
copying cadquery/occ_impl/exporters/assembly.py -> build/lib/cadquery/occ_impl/exporters
copying cadquery/occ_impl/exporters/vtk.py -> build/lib/cadquery/occ_impl/exporters
copying cadquery/occ_impl/exporters/json.py -> build/lib/cadquery/occ_impl/exporters
running egg_info
creating cadquery.egg-info
writing manifest file 'cadquery.egg-info/SOURCES.txt'
writing manifest file 'cadquery.egg-info/SOURCES.txt'
copying cadquery/py.typed -> build/lib/cadquery
running bdist_wheel
<home folder>/.virtualenvs/wasp/lib/python3.9/site-packages/setuptools/command/install.py:34: SetuptoolsDeprecationWarning: setup.py install is deprecated. Use build and pip and other standards-based tools.
  warnings.warn(
running install
running install_lib
creating build/bdist.emscripten_wasm32
creating build/bdist.emscripten_wasm32/wheel
creating build/bdist.emscripten_wasm32/wheel/cadquery
creating build/bdist.emscripten_wasm32/wheel/cadquery/plugins
creating build/bdist.emscripten_wasm32/wheel/cadquery/contrib
creating build/bdist.emscripten_wasm32/wheel/cadquery/occ_impl
creating build/bdist.emscripten_wasm32/wheel/cadquery/occ_impl/importers
creating build/bdist.emscripten_wasm32/wheel/cadquery/occ_impl/exporters
running install_egg_info
running egg_info
writing manifest file 'cadquery.egg-info/SOURCES.txt'
Copying cadquery.egg-info to build/bdist.emscripten_wasm32/wheel/cadquery-2.1-py3.9.egg-info
running install_scripts
Unpacking to: ./cadquery-2.1...OK
Repacking wheel as ./cadquery-2.1-py3-none-any.whl...OK
[2022-01-30 13:11:35] Succeeded building package cadquery in 6.7 seconds.
```

```
$ ls build/ dist/
build/:
cadquery-2.1		cadquery-master.zip

dist/:
cadquery-2.1-py3-none-any.whl
```

`build/cadquery-2.1/build/bdist.emscripten_wasm32/` is empty

### Environment

pyodide commit 75aa9ba678e0a8733259fea3cc3e4c786e12fab7
building on mac with ./run_docker

**Comment by rth** [comment]
Thanks for the report!

Yes, as far as I can tell the current version of the documentation is not correct, because it assumes you already have Pyodide built. We need to fix those docs.

For now, you can use,
```
PYODIDE_PACKAGES="cadquery" make
```
to build your package.

**Comment by hoodmane** [comment]
You can see from the log that @paulftw is using a branch with #2027. The problem here is that we didn't update the docs after #2027.
Instead of:
```sh
cp build/*.data build/*.js ../../build/
```
it should say:
```sh
cp dist/*.whl ../../build
```

**Comment by paulftw** [comment]
Thanks, @rth! making progress in that direction now.

**Comment by impermaxvict** [comment]
Hi @paulftw, I just wanted to mention that I am currently also attempting to build CadQuery/cadquery#936.
You may [find my efforts here](https://github.com/impermaxvict/cadquery-pyodide-port), however, I am still struggling to build [OCP](https://github.com/CadQuery/OCP).

**Comment by ryanking13** [comment]
We've [updated our outdated documentation](https://pyodide.org/en/latest/development/new-packages.html). I'll close this as resolved. If you still have an issue with building your package. Please open a new issue.

**Comment by koodistrom** [comment]
@paulftw Did you manage to create a CadQuery Pyodide package? I came across this issue when googling for a way to run CQ in browser.
